src/products/views/ProductList/ProductList.tsx

```diff
@@ -223,14 +232,30 @@ export const ProductList: React.FC<ProductListProps> = ({ params }) => {
           id: data.exportProducts.exportFile.id,
         });
         closeModal();
-        reset();
+        clearRowSelection();
       }
     },
   });
 
+  const [productBulkDelete, productBulkDeleteOpts] =
+    useProductBulkDeleteMutation({
+      onCompleted: data => {
+        if (data.productBulkDelete.errors.length === 0) {
```

Comment:
**question**: do we have error handling? or we are covered by global error handling?

Reply:
Yes it will be covered globally